CSharpSourceCode/TOW_Core.csproj

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
   <!-- Bannerlord path Property-->
   <PropertyGroup>
-    <GameDir>C:\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord</GameDir>
+    <GameDir>A:\Steam\steamapps\common\Mount &amp; Blade II Bannerlord TOW</GameDir>
```

Comment:
ARRRGG dont push that. all of us have to go through the pain and set this back :D

Reply:
As for me everything is fine except for this line